common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -1223,7 +1224,7 @@ def test_if_captions_are_draggable(self):
             )
         else:
             self.assertEqual(
-                captions_end.get('y') + 15,
+                captions_end.get('y') + 16,
```

Comment:
@Qubad786 the context was I got the same kind of failure, and so I changed the number! It seems that these drag-and-drop tests are very pixel sensitive. My assumption was that somehow the fonts were slightly different, which caused the drag targets to be slightly off. I don't see anything in your code that would change how these targets would render, but if they reliably pass with the updated number then by all means change it back.